yaml/deployment.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: minio-kv
+  labels:
+    app: minio-kv
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: minio-kv
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      name: minio-kv
+      labels:
+        app: minio-kv
+    spec:
+      volumes:
+      - name: data
+        persistentVolumeClaim:
```

Comment:
Most folks using this won't have a PV available.

Can we move Minio into a separate YAML file please?

Reply:
Minio I believe is already available on helm anyway?

Reply:
i.e. https://github.com/openfaas-incubator/ofc-bootstrap/blob/master/scripts/install-minio.sh